src/Evolution/Executables/GeneralizedHarmonic/EvolveGeneralizedHarmonicWithHorizon.hpp

```diff
@@ -134,7 +134,7 @@ struct EvolutionMetavars<3, InitialData, BoundaryConditions>
   };
 
   using const_global_cache_tags = tmpl::list<
-      typename gh_base::analytic_solution_tag,
+      typename gh_base::analytic_solution_tag, observers::Tags::SurfaceFileName,
```

Comment:
You've already included this tag in the `ObserveSurfaceData` callback. Do you really need to include it here?

Reply:
The executables failed to compile, complaining of missing tags, unless I included them in the executables as well